script.js

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable no-use-before-define */
 /* eslint-disable class-methods-use-this */
```

Comment:
- [ ] Let's avoid disabling linters. Always try to find out how to fix the linter errors instead of disabling the linters. In this case, I'll leave some hints for you in the following code comments, but please try to do this on you own next time, or ask for help from peers if you run into trouble.